solr/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # syntax=docker/dockerfile:1
 FROM solr:latest
 COPY ./jts-core-1.15.0.jar /opt/solr/server/solr-webapp/webapp/WEB-INF/lib/
-# uncomment this if we want to run a solr replication cluster with local zookeeper
-# COPY ./zoo.cfg /opt/solr/server/solr/zoo.cfg
\ No newline at end of file
+# this exposes the embedded zookeeper on the Docker network, so other services can talk to it (like the prometheus exporter)
+COPY ./zoo.cfg /opt/solr/server/solr/zoo.cfg
```

Comment:
As stated previously, we need to expose zookeeper so that `solr_exporter` can publish the solr stats.